src/dotnet/CommandLine/CommandLineApplication.cs

```diff
@@ -372,11 +372,11 @@ public void ShowHelp(string commandName = null)
                 if (cmd != this && cmd.Arguments.Any())
                 {
                     var args = string.Join(" ", cmd.Arguments.Select(arg => arg.Name));
-                    headerBuilder.Insert(usagePrefixLength, string.Format(LocalizableStrings.UsageItemWithArgs, cmd.Name, args));
+                    headerBuilder.Insert(usagePrefixLength, string.Format(" {0} {1}", cmd.Name, args));
```

Comment:
These templates were originally in LocalizableStrings to allow for string ordering reversals [RTL languages, etc]. Is that no longer needed?